dev/common/setup_test_data.py

```diff
@@ -55,6 +61,12 @@
 admin.is_staff = True
 admin.save()
 
+# Note: this user is not a part of ephemeral keycloak users
+ee_admin, _ = User.objects.get_or_create(username="ee-admin")
+ee_admin.set_password("redhat")
+ee_admin.groups.add(ee_group)
+ee_admin.save()
```

Comment:
If we ever need a new test user that will also be tested in insights mode, we will want to pick a username that is already in keycloak so the test users are consistent in standalone and insights mode. For this execution environment user, I think it is ok to not match up to keycloak.